mava/components/training/step.py

```diff
@@ -284,7 +284,7 @@ def get_behavior_values(
 
             # Denormalise the values here to keep the GAE function clean
             target_value_stats = states.target_value_stats
-            if hasattr(trainer.store, "target_running_stats_fn"):
+            if trainer.store.global_config.normalize_target_values:
```

Comment:
I like this a lot more than the hasattr :raised_hands:

Reply:
Cool